jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildImageTask.java

```diff
@@ -133,6 +134,9 @@ public void buildImage()
           "container.workingDirectory is not an absolute Unix-style path: "
               + ex.getInvalidPathValue(),
           ex);
+    } catch (InvalidContainerVolumeException ex) {
+      throw new GradleException(
+          "container.volumes is not an absolute Unix-style path: " + ex.getInvalidVolume());
```

Comment:
Sorry for catching this late. Let's propagate the exception. Here and the other 5 places.

```java
throw new GradleException("..." + ex.getInvalidVolume(), ex);
```

Reply:
No problem at all. Added the cause to all places that wrap this exception.